app/events/db/db_table_field_mappings.yaml

```
@@ -792,4 +792,35 @@ mappings:
         unique_key: iiq_id
         fields:
             id: iiq_id
-            status: status
\ No newline at end of file
+            status: status
+
+    audience_overlap_request:
+        name: audience_overlap
+        schema: iiq_schema
+        unique_key: id
+        fields:
+            id: id
```

as a convention this has been mapped to `iiq_id` correct?
```suggestion
            id: iiq_id
```